litellm/llms/bedrock.py

```diff
@@ -504,12 +513,12 @@ def init_bedrock_client(
     aws_session_name: Optional[str] = None,
     aws_profile_name: Optional[str] = None,
     aws_role_name: Optional[str] = None,
+    extra_headers: Optional[dict] = None,
```

Comment:
@themrzmaster we already have support for something similar via metadata, see how langfuse logging works - https://docs.litellm.ai/docs/observability/langfuse_integration#set-custom-trace-id-trace-user-id-and-tags

couldn't we just use the metadata field here?

Reply:
sounds like the purpose of extra headers is for bedrock logging?

Reply:
in my scenario, I use bedrock over a custom proxy. i need to pass custom authorization header to be able to access the endpoint